### DISCUSSION TITLE
Compatibility with ALL MD editors: Outline <-> Github Synchronization

### DISCUSSION THREAD

**Discussion by ReessKennedy** [outline/outline#5375] [Ideas]
My idea: What if Outline could send page changes to a git repository? 

This would give Outline some of the powers built into Gitbook (https://docs.gitbook.com/product-tour/git-sync/enabling-gitlab-sync) and expand utility for developers that love just working directly with markdown files in VS code while still allowing less technical folks, or devs that don't like editing markdown files directly, the option to use the WYSIWYG while having everything SYNCHRONIZED no matter which tool you use. 

More background: 
There seems be an explosion of interest in and adoption of markdown editors: Obsidian, Logseq, Taio, Typora, Zettlr. Obsidian and Logseq, especially have huge communities now. 

I, personally, moved all my personal notes to Obsidian and have been loving it. Team notes still live on Outline but everyday I find something in my personal notes that I think should be moved to Outline and it becomes an annoying amount of friction. Some sort of GIt sync would solve this.

**Comment by tommoor** [comment]
Definitely open to an `Outline -> GitHub` integration but the other direction isn't likely to work reliably as converting a document to Markdown is lossy, not everything can be represented in Markdown – in fact it's quite limited.

**Reply by ReessKennedy** [reply]
Right. Yeah, I suspected that one way sync would be much simpler. Even this would help. Changes could still be done by individuals on their local machine in the markdown editor of their choice and even committed to a branch and then at the end of the day they could just update the important bits in the team docs in Outline. They'd at least be able to use the search within their local and receive all relevant text matches but always having the latest text from Outline synced through Github.

**Reply by ReessKennedy** [reply]
Continued great work on Outline @tommoor !
